modules/afsocket/transport-mapper-inet.h

```diff
@@ -32,26 +32,21 @@ typedef struct _TransportMapperInet
 
   gint server_port;
   const gchar *server_port_change_warning;
-  guint32 flags;
-  gboolean require_tls;
-  gboolean allow_tls;
-  gboolean require_tls_when_has_tls_context;
+  /* tls() options are required */
+  gboolean require_tls_configuration;
+  /* tls() options are optional, but are permitted */
+  gboolean allow_tls_configuration;
+  gboolean delegate_tls_start_to_logproto;
```

Comment:
Could you please move the comments `the proto can start TLS but we won't do that ourselves` and `it is up to the transport plugin to start TLS` here?

Reply:
I added a more elaborate comment but left the originals there as well.